dashboard/config/videos.csv

```diff
@@ -121,10 +121,15 @@ tutorial_puzzle_challenge,,,_l68chtl1co,http://videos.code.org/2014/tutorial-puz
 infinity_playlab_intro,,,vQGOfhsb76o,http://videos.code.org/2015/infinityplaylab/infinity-playlab-intro.mp4
 infinity_playlab_events,,,HcljJx_0Elw,http://videos.code.org/2015/infinityplaylab/infinity-playlab-events.mp4
 infinity_playlab_repeat_forever,,,Ldj1-kEa9nA,http://videos.code.org/2015/infinityplaylab/infinity-playlab-repeat-forever.mp4
-csd_concept_computer,,,f7WfMGA4u_w,http://videos.code.org/cs-discoveries/concept/what_makes_a_computer.mp4
-csd_concept_binary_data,,,NCi7BQ6OiCE,http://videos.code.org/cs-discoveries/concept/binary_and_data.mp4
+csd_concept_how_computers_work,,,OAx_6-wdslM,http://videos.code.org/cs-discoveries/concept/01_HowComputersWork_sm.mp4
+csd_concept_computer,,,mCq8-xTH7jA,http://videos.code.org/cs-discoveries/concept/02_WhatMakesAComputer_sm.mp4
```

Comment:
csd_concept_computer and csd_concept_circuits_logic are meant to have new videos, but not new titles?

Reply:
Yup, we had draft versions up for use in the curriculum.